addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -6543,182 +6553,243 @@ stock GetBossIndex(client)
 	return -1;
 }
 
-stock CalcBossHealthMax(client)
+stock Calculate(client, const String:key[], const String:defaultFormula[], malformDefault)
 {
 	decl String:formula[1024], String:buffer[2];
-	new String:value[1024];
-	new bool:mustClose, bool:usePlayers, bool:canAdd, bool:valueReady;
-	new parentheses;
-	new Float:sum[128];
-	new _operator[128];
-
-	KvRewind(BossKV[Special[client]]);
-	KvGetString(BossKV[Special[client]], "health_formula", formula, sizeof(formula), "(((760.8+n)*n-1)^1.0341)+2046");
+	new String:value[1024], Float:sum[128], calcOperators:_operator[128];
+	new bool:closeBracket, bool:usePlayers, bool:useLives, bool:foundOperator, bracket;
+	new Handle:bKv = BossKV[Special[client]];
```

Comment:
Can we reference this directly without creating a new variable for it?  That's how it's done everywhere else in the code.